docs/internal/RunDocumentationManjaro.md

```diff
@@ -0,0 +1,69 @@
+# Setting up MkDocs on Manjaro 
+
+This is a step-by-step guideline for installing and running the MkDocs Documentation on Manjaro Linux distribution.
+
+## MkDocs Installation
+
+As you may already know, MkDocs is Python-based. Therefore, you need to install a recent version of Python as well as its package manager, `Pip`. Luckily, both of these are installed by default on Manjaro (like on most Linux distros). If you are using the latest version of Python, then Pip is definitely installed by default.
+
+You may, however, need to upgrade Pip to latest version if it's not up-to-date.
+
+```
+pip install --upgrade pip
+```
+To check for the version of Python and Pip you are using:
+
+```
+python --version
+pip --version
+```
+You can now install MkDocs using Pip:
+
+```pip install mkdocs
+```
+
+## Install Dependencies
+
+```
+pip install -r requirements.txt
+pip install -r python_scripts/requirements.txt
+```
+
+Clone `Lexer`, a Tremor-specific syntax highlighting tool into Tremor:
+```
+.github/scripts/install-lexer.sh
+```
+Install CMake, an open-source, cross-platform family of tools designed to build, test and package software. CMake will help in the compilation process, and to generate native makefiles.
+
+!!! Use `Pacman`, the package manager for Manjaro, for the installation from root:
+```
+sudo pacman -S cmake
+```
+
+## Generate Dynamic Documentation and Configuration
```

Comment:
I think this needs to move to before `Local Doc Service` since we need the dependencies first.

Reply:
Do you mean that `Building Documentation` should come before `Generating Dynamic Documentation and Configuration`?